### PR TITLE
avoid crash on corrupt footer

### PR DESCRIPTION
**Pull request by mcvain**
equivalent of https://github.com/xdf-modules/xdf-Matlab/pull/19 but for pyxdf
closes https://github.com/labstreaminglayer/App-LabRecorder/issues/107

### CODE REVIEW

File: pyxdf/pyxdf.py [hunk 1]
```diff
@@ -328,7 +328,16 @@ def load_xdf(
...
+                try:
+                    streams[StreamId]["footer"] = _xml2dict(fromstring(xml_string))
+                except Exception as e:
+                    if type(e).__name__ == 'ParseError':
+                        logger.error(
+                            "found likely XDF file corruption (%s), "
+                            "ignoring corrupted XML element in footer." % e
+                        )
+                    else:
+                        raise
```
Comment (cbrnr) [inline pyxdf/pyxdf.py:340]: Can you simplify this by catching only a `ParseError` exception in the first place? No need for the `if`/`else` clause then. Thanks!

Comment (mcvain) [reply pyxdf/pyxdf.py]: Okay. Thank you!

Comment (cbrnr) [reply pyxdf/pyxdf.py]: One last nitpick, would you mind using an f-string?

Comment (mcvain) [reply pyxdf/pyxdf.py]: I am happy to, I just thought I should be consistent with the other error messages. Let me know if it looks good!

**Comment by cbrnr** [comment]
Thanks a lot @mcvain!

**Pull request merged**